### ISSUE TITLE
Improve Accessibility of Language Select Boxes

### ISSUE THREAD

**Issue by jpdevries** [modxcms/revolution#12793]
### Summary

As you can notice towards the end of [this screencast](http://j4p.us/0v1I1t17203e) the languages in the language select box aren't read accessibly.
### Step to reproduce

Navigate to the language select box using a screen reader.
### Observed behavior

Languages aren't decipherable. 
### Expected behavior

Languages should be decipherable. 
### Environment

MODX 2.4 Mac OS X Safari with VoiceOver

**Comment by Mark-H** [comment]
I assume based on our chats that "decipherable" really means "needs to be the full language name"?

**Comment by jpdevries** [comment]
Yes. That would be great!

**Comment by alroniks** [comment]
It was improved and the list of languages can be listened by voice over but the list of languages contains languages in the native sounds that may confuse listeners. Probably need to add some attributes with value for reading on the system language.

**Comment by OptimusCrime** [comment]
@Alroniks What language would otherwise make sense? To have the languages in their own language?

**Comment by alroniks** [comment]
Yes, I think we can detect user language and add attributes with language name on this language. Something like that when user language is English:
```
<option value="ru" data-language="Russian">Русский</option>
```

**Comment by alroniks** [comment]
Fixed
